Makefile

```diff
@@ -80,6 +80,14 @@ rpms: images
 	docker cp $$(docker create $(IMAGE)/centos7rpmbuild):/data/.rpms .
 	docker rm $$(docker ps -aq) -f
 
+copr-build: rpms
+	mkdir -p .srpms
+	rm -frv .srpms/*
+	docker cp $$(docker create $(IMAGE)/centos8rpmbuild):/data/.srpms .
+	docker cp $$(docker create $(IMAGE)/centos7rpmbuild):/data/.srpms .
+	docker rm $$(docker ps -aq) -f
+	copr-cli --config .copr.conf build --nowait zhukovgreen/convert2rhel .srpms/*
```

Comment:
@bocekm can we build right into `@oamg/convert2rhel`? What do we need for that? Adding my user into the oamg org under copr?

Reply:
It's ok to use `@oamg/convert2rhel`. The only restriction is to not build packages with a bumped [release](https://github.com/oamg/convert2rhel/blob/v0.22/packaging/convert2rhel.spec#L13).  And yes, each user who want's to build in that copr repo needs to ask for permission under https://copr.fedorainfracloud.org/coprs/g/oamg/convert2rhel/permissions/.